zest/releaser/utils.py

```diff
@@ -88,7 +168,10 @@ def get_next_answer(self):
 def get_input(question):
     if not TESTMODE:
         # Normal operation.
-        return raw_input(question).strip()
+        result = input(question)
```

Comment:
This is something that needs fixing. input() on python2 *evaluates* what you put in. See http://stackoverflow.com/a/15129556/27401

If I enter `1.2rc` as a version string, python evaluates it under 2.x and gives a syntax error.